src/World.h

```diff
@@ -454,7 +454,7 @@ class cWorld :
 
 	/** Spawns a boat at the given coordinates.
 	Returns the UniqueID of the spawned boat, or cEntity::INVALID_ID on failure. */
-	UInt32 SpawnBoat(double a_X, double a_Y, double a_Z, cBoat::eMaterial a_Material);
+	UInt32 SpawnBoat(Vector3d a_Pos, cBoat::eMaterial a_Material);
```

Comment:
This is exported to lua, you'll either need to keep both as overloads or add a manual binding that accepts both forms.